apps/pdc-frontend/package.json

```diff
@@ -36,6 +36,8 @@
     "@utrecht/component-library-react": "5.0.0",
     "@utrecht/design-tokens": "2.4.0",
     "@utrecht/web-component-library-react": "1.3.0",
+    "@utrecht/open-forms-container-react": "1.0.0",
+    "@utrecht/open-forms-container-css": "1.0.0",
```

Comment:
Is `@utrecht/open-forms-container-css` nodig? Gaat dat niet automatisch via de React package?

Reply:
Ik heb het probleem in de beschrijving uitgelegd.
https://github.com/frameless/strapi/pull/1125#issue-2982802192

Reply:
Op basis van het gesprek met @Robbert  over dit probleem is de conclusie dat er een issue-ticket voor aangemaakt moet worden en dat deze PR niet geblokkeerd hoeft te worden.

Zodra dit probleem-[#1132](https://github.com/frameless/strapi/issues/1132) is opgelost, moeten we dit probleem opnieuw oppakken.

Reply:
https://github.com/frameless/strapi/issues/1132#issuecomment-2820373305